python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -670,16 +692,23 @@ def key(self) -> AssetKey:
     @public
     @property
     def resource_defs(self) -> Mapping[str, ResourceDefinition]:
+        """Mapping[str, ResourceDefinition]: A mapping from resource name to ResourceDefinition for
+        the resources bound to this AssetsDefinition.
+        """
         return dict(self._resource_defs)
 
     @public
     @property
     def keys(self) -> AbstractSet[AssetKey]:
+        """AbstractSet[AssetKey]: The asset keys associated with this AssetsDefinition."""
         return self._selected_asset_keys
 
     @public
     @property
     def dependency_keys(self) -> Iterable[AssetKey]:
+        """Iterable[AssetKey]: The asset keys which are upstream of assets associated with this
+        AssetsDefinition.
+        """
```

Comment:
nit: I think it could be a bit clearer-- maybe "upstream of any asset included in this `AssetsDefinition`"